app.py

```diff
@@ -356,10 +356,13 @@ def main():
             # Make prediction
             predicted_score = model.predict(input_data_scaled)[0]
 
+            # Check if predicted score is greater than 5
+        if predicted_score > 5:
+            st.error("Uh-Oh, it seems that the combination of your inputs is very unique and our model is having trouble taking into account this unique combo. Try something else!")

```

Comment:
If the score ends up higher than 5, then can we limit it to 5. Logic is that the inputs of their property are really good and we predict the score to be 100%.